release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -105,6 +105,10 @@ The {product-title} internal registry and image streams now support Open Contain
 [id="ocp-4-7-machine-api"]
 === Machine API
 
+[id="ocp-4-7-machine-api-wait-upgrade"]
+==== Updates are immediately blocked if a machine config pool is degraded
+
+If a machine config pool (MCP) is in a `degraded` state, the Machine Config Operator (MCO) now reports its *Upgradeable* status as *False*. As a result, you are now prevented from performing an update within a minor version, for example, from 4.7 to 4.8, until all machine config pools are healthy. Previously, with a degraded machine config pool, the Machine Config Operator did not report its *Upgradeable* status as *false*. The update was allowed and would eventually fail when updating the Machine Config Operator because of the degraded machine config pool. There is no change in this behavior for updates within z-stream releases, for example, from 4.7.1 to 4.7.2. As such, you should check the machine config pool status before performing a z-stream update.
```

Comment:
s/you should check/you must check

Since the MCO acronym is provided, should subsequent mentions of the Machine Config Operator be changed to MCO?

Reply:
@bmcelvee I used MCO and MCP in several places but it seemed like acronym soup. It seems to read better with the terms spelled out (but left the parenthetical acronyms for folks who know the components that way).  I can change back if you feel stongly.